wremnants/theory_tools.py

```diff
@@ -19,11 +22,92 @@
 
 scale_tensor_axes = (axis_muRfact, axis_muFfact)
 
-pdfMap = {"nnpdf31" : {
+pdfMap = {
+        "nnpdf31" : {
             "name" : "pdfNNPDF31",
             "branch" : "LHEPdfWeight",
-            },
-        }
+            "combine" : "symHessian",
+            "entries" : 101,
+            "alphas" : ["LHEPdfWeight[101]", "LHEPdfWeight[102]"],
+        },
+        "ct18" : {
+            # This has CT18 + CT18Z in it :-/
+            "name" : "pdfCT18",
+            "branch" : "LHEPdfWeightAltSet18",
+            "combine" : "asymHessian",
+            "entries" : 58,
+            "alphas" : ["LHEPdfWeightAltSet18[59]", "LHEPdfWeightAltSet18[60]"],
+			"alphaRange" : "002",
```

Comment:
Where can I double check the mapping of these to LHAPDF ids?

Reply:
In the NanoAOD meta data, with Events->Print("*PdfWeight*"). 

For the new PDFs its a bit of a disaster, since the sets are read from LHAPDF, and 10_6 has an older version, so they often get grouped together and mis-identified. I would say that double checking me on this is probably not worth it right now.